.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,43 @@
+name: helm-package
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'charts/kyverno/**'
+env:
+  HELM_CHART_URL: "https://nirmata.github.io/kyverno/helm-charts/"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+    - name: Install Helm
+      run: |
+        curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get
+        chmod 700 get_helm.sh
+        ./get_helm.sh
+        helm init --client-only
+    - name: Check Chart Version
```

Comment:
Looking at this check that is being done when pushing to master, this means that the build of the chart will fail after a new commit is pushed to master.
I would suggest that this check is done on PR branche before the merge is done to make sure that everything is fine before the merge.

Reply:
Yes.....You should update the chart version whenever you are making any changes in the chart